src/lib/features/personal-dashboard/personal-dashboard-service.ts

```diff
@@ -117,11 +123,24 @@ export class PersonalDashboardService {
         const owners =
             await this.projectOwnersReadModel.getProjectOwners(projectId);
 
+        const allRoles = await this.accessStore.getAllProjectRolesForUser(
```

Comment:
in the next PR we can Promise.all all those async operations

Reply:
Ah, yeah. Actually lemme do it right away, while I'm in that context.

Reply:
Nah, I'll do it in a follow-up after all. Think it's better served that way